primitives/src/targeting/eval.rs

```diff
@@ -196,8 +171,8 @@ impl TryFrom<SerdeValue> for Value {
         match serde_value {
             SerdeValue::Bool(bool) => Ok(Self::Bool(bool)),
             SerdeValue::Number(number) => Ok(Self::Number(number)),
-            // It's impossible to have a BigNumber literal in the rules, since they're JSON based (conform to serde_json::value::Value)
-            // However it is possible to obtain a BigNumber by invoking the Function::Bn
+            // It's impossible to have a UnifiedNum literal in the rules, since they're JSON based (conform to serde_json::value::Value)
+            // However it is possible to obtain a UnifiedNum by invoking the Function::Bn
```

Comment:
Can you still obtain UnifiedNum with `Function::Bn`?

Reply:
Yes, the `UnifiedNum` is deserialized the same way as the `BigNum` in JSON - a string.